rotkehlchen/types.py

```diff
@@ -1484,6 +1484,11 @@ class OracleSource(SerializableEnumNameMixin):
 )
 
 
+class EventMetricKey(StrEnum):
+    """Keys for metrics stored in the event_metrics table."""
+    BALANCE = 'balance'
+
```

Comment:
use `SerializableEnumNameMixin` . it overloads the str method but call serialize when used since it is our way to detect usage and refactor later